src/org/omegat/util/Preferences.java

```diff
@@ -215,7 +217,8 @@ public final class Preferences {
     public static final String MARK_TRANSLATED_SEGMENTS = "mark_translated_segments";
 
     public static final String MARK_AUTOPOPULATED = "mark_autopopulated";
-
+    public static final boolean MARK_AUTOPOPULATED_DEFAULT = true;
```

Comment:
@brandelune when you want to define a new default boolean in Preferences class, why not set ALL the defaults?
There are many direct value in `restoreDefaults` methods.

Reply:
Because I wanted to modify only the minimum set.

We discussed with @t-cordonnier that the way preferences were set in OmegaT would require a major overhaul, but I don't have the skills to do that (I barely understood what I was doing this time...)

It's something I'd be interested in working on, but not at the moment.

Reply:
OK, I can understand it and will work on it in 6.1 versions.